### ISSUE TITLE
Temporarily disable notification for programs opened outside a box when forced programs are disabled

### ISSUE THREAD

**Issue by bastik-1001** [sandboxie-plus/Sandboxie#1182]
**Is your feature request related to a problem? Please describe.**
Right now (at least up to 0.9.5) when disabling forced programs, for 60 seconds or so, and then opening an executable Sandboxie-Plus issues a notification that said program was started outside the sandbox.

It is desired to have the notification being issued, as programs are excepted to run sandboxed and when they fail or erroneously get started outside a box  I wanted to get to know about it, just not when I expect the programs to be launched outside a box as I disabled them being forced into a box.

**Describe the solution you'd like**
Temporarily disable notification for programs opened outside a box when forced programs are disabled.

So while forced programs are disabled for the period of time specified by the user, Sandboxie-Plus should disable or suppress the notifications of forced programs being opened outside the sandbox.

**Comment by DavidXanatos** [comment]
sounds reasonable, but how does the clasic build behave in that scenario?

**Comment by bastik-1001** [comment]
I am going to assume that SbieCtrl.exe in Sandboxie Classic installations behaves in the same was as in the plus installation. As I launched SbieCtrl.exe from my plus installation it notified me that I launched a program outside the sandbox after I disabled forced programs and stated a previously forced program.

While I consider the classic version to be legacy, they should still function similarly, So it's up to you to decide whenever it is worth to bring that feature to both versions or just the Plus version.
